bin/kano-updater-quickcheck

```diff
@@ -5,12 +5,27 @@
 # Copyright (C) 2015 Kano Computing Ltd.
 # License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
 #
+# Runs the updater in boot-window mode, if upgrades are avaiable.
+# Pass "--dry-run" to simply get an errorcode indication, 1 means updates are available.
+#
 
 STATUS_FILE=/var/cache/kano-updater/status.json
+dry_run=$1
 
-# check if we need to run 
-DO_RUN=$(jq '.is_scheduled!=0 or .state!="no-updates"' $STATUS_FILE)
+function are_updates_available()
+{
+    # check if we need to run
+    DO_RUN=$(jq '.is_scheduled!=0 or .state!="no-updates"' $STATUS_FILE)
+    if [ "$DO_RUN" == "true" ]; then
+        return 1
```

Comment:
`Functions return a value, called an exit status. This is analogous to the exit status returned by a command. The exit status may be explicitly specified by a return statement, otherwise it is the exit status of the last command in the function`

 * http://tldp.org/LDP/abs/html/complexfunct.html

You can also use a variable to return more complex things, as you say.